apps/settings/js/modules/panel_utils.js

```diff
@@ -30,10 +30,10 @@ define(function(require) {
 
       request.onsuccess = function() {
         switch (input.type) {
-          case 'radio':
+          case 'gaia-radio':
             input.checked = (input.value == request.result[key]);
             break;
-          case 'checkbox':
+          case 'gaia-checkbox':
```

Comment:
Have we removed all checkbox? if not we can't just change the cases yet. Please add another gaia-checkbox case instead